extensions/amp-brightcove/0.1/amp-brightcove.js

```diff
@@ -456,8 +460,8 @@ class AmpBrightcove extends AMP.BaseElement {
   }
 
   /** @override */
-  play(unusedIsAutoplay) {
-    this.sendCommand_('play');
+  play(isAutoplay) {
+    this.sendCommand_('play', isAutoplay);
```

Comment:
Just for my own understanding: what's the downstream effect of passing `isAutoplay`?

Reply:
Some integrations within the player need to distinguish an autoplay, e.g. to change params in an ad call.